common/utils/works.ts

```diff
@@ -121,20 +122,16 @@ export function getHoldings(work: Work): Holding[] {
   return work?.holdings || [];
 }
 
-export function getItemsWithPhysicalLocation(
-  work: Work
-): PhysicalItemAugmented[] {
+export function getItemsWithPhysicalLocation(work: Work): PhysicalItem[] {
   return (work.items ?? [])
     .map(item => {
       if (
         item.locations.find(location => location.type === 'PhysicalLocation')
       ) {
-        return item as PhysicalItemAugmented;
+        return item as PhysicalItem;
       }
     })
-    .filter((item?: PhysicalItemAugmented): item is PhysicalItemAugmented =>
-      Boolean(item)
-    );
+    .filter((item?: PhysicalItem): item is PhysicalItem => Boolean(item));
```

Comment:
We could create some [type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) here

```ts
function isPhysicalItem(item: Item): item is PhysicalItem {
  return Boolean(item.locations.find(location => location.type === 'PhysicalLocation'))
}
```